src/lib/sessionStorage.js

```diff
@@ -0,0 +1,12 @@
+export const getFromSessionStorage = (key) => {
```

Comment:
Rewrite to TS. An example how to type it properly with generic:
https://github.com/Adamant-im/adamant-im/blob/master/src/lib/nodes/storage.ts#L17